themes/eds/views/showcase.html

```diff
@@ -41,7 +41,7 @@
                     <ul class="dropdown-menu">
                       {% for item in dataset.downloads %}
                       <li>
-                        <a href="{{ item.path }}" class="data-extract-btn" data-format="{{ item.format }}">{{ item.format }}</a>
+                        <a href="{{ item.path }}" class="data-extract-btn" data-format="{{ item.format }}"  onclick="sendGAEvent('{{ dataset.name }}', '{{ dataset.resources[loop.index0].name }}', '{{ item.format }}')">{{ item.format }}</a>
```

Comment:
This won't work because `dataset.downloads` isn't the same as `dataset.resources` - it looks like this:

```
[
      {
        format: 'CSV',
        path: datapackage.resources[0].path
      },
      {
        format: 'TSV',
        path: datapackage.resources[0].path + '?format=tsv'
      },
      {
        format: 'XLSX - Excel',
        path: datapackage.resources[0].path + '?bom=true'
      },
      {
        format: 'JSON',
        path: datapackage.resources[0].path + '?format=json'
      },
      {
        format: 'XML',
        path: datapackage.resources[0].path + '?format=xml'
      }
    ]
```

The thing is that in EDS there is a convention to have 1 resource per 1 dataset so no need to do `dataset.resources[loop.index0].name`, instead, you can assume there is only 1 resource so `dataset.resources[0].name`.

I will push a commit for this.